stubs/python-jose/jose/jwt.pyi

```diff
@@ -1,16 +1,25 @@
+from collections.abc import Container, Iterable, Mapping
 from typing import Any
 
-def encode(claims, key, algorithm=..., headers: Any | None = ..., access_token: Any | None = ...): ...
+from .backends.base import Key
+
+def encode(
+    claims: Mapping[str, Any],
```

Comment:
For this function, the `claims` dictionary is mutated inside the function, so it does need to be a `MutableMapping` rather than just a `Mapping`

```suggestion
    claims: MutableMapping[str, Any],
```